### PR TITLE
Allow the beats inputs to bind to a specific host

### PR DESCRIPTION
**Pull request by ph**
The host options was not correctly send to the server code.

Fixes: #146

### CODE REVIEW

File: spec/support/shared_examples.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@
     expect(subject.get("tags")).to include(tag)
   end
 
-  context "when the `beast.hotname` doesnt exist on the event" do
+  context "when the `beast.hostname` doesnt exist on the event" do
```
Comment (talevy) [inline spec/support/shared_examples.rb]: s/beast/beats

File: src/main/java/org/logstash/beats/Server.java [hunk 2]
```diff
@@ -29,21 +29,27 @@
...
     private final int port;
     private final NioEventLoopGroup bossGroup;
     private final NioEventLoopGroup workGroup;
+    private final String host;
     private IMessageListener messageListener = new MessageListener();
     private SslSimpleBuilder sslBuilder;
 
     private final int clientInactivityTimeoutSeconds;
 
     public Server(int p) {
```
Comment (talevy) [inline src/main/java/org/logstash/beats/Server.java]: is this constructor ever used? maybe we can go so far as to force an explicit host at all times and fall back to the `@host` used in the logstash plugin config

Comment (ph) [inline src/main/java/org/logstash/beats/Server.java]: true, I will remove it for simplicity sake.

**Comment by ph** [comment]
I've made the requested changes.

File: src/main/java/org/logstash/beats/Server.java [hunk 3]
```diff
@@ -29,21 +29,23 @@
 
     static final long SHUTDOWN_TIMEOUT_SECONDS = 10;
     private static final int DEFAULT_CLIENT_TIMEOUT_SECONDS = 15;
-
+    private static final String DEFAULT_HOST = "0.0.0.0";
```
Comment (talevy) [inline src/main/java/org/logstash/beats/Server.java]: not used

Comment (ph) [inline src/main/java/org/logstash/beats/Server.java]: done

**Comment by talevy** [comment]
LGTM

**Pull request closed without merging**